### ISSUE TITLE
Readline support in faktory-cli

### ISSUE THREAD

**Issue by mperham** [contribsys/faktory#105]
It would be really nice to have support for arrow key functionality within the CLI, history, edit by word, jump to start or end of line, etc.  Integrate https://github.com/chzyer/readline or another similar package.

For example, run `make repl` today, type `help` and then hit the Up arrow, notice how you get `> ^[[A `, not the previous command filled in for you:

```
$ make repl
go generate github.com/contribsys/faktory/webui
go run cmd/faktory-cli/repl.go -l debug -e development
Faktory 0.6.1
Copyright © 2017 Contributed Systems LLC
Licensed under the GNU Public License 3.0
Using RocksDB 5.8.0 at /Users/mike/.faktory/db
> help
[snip]
> ^[[A    
```

**Comment by ustrajunior** [comment]
Hey, Mike. I can work on this one. I will start and then I talk to you to check the mappings we can use for each action, ok?

**Comment by mperham** [comment]
Sounds rad, excited to see what you come up with!

**Comment by mperham** [comment]
Fixed by #109
